app/src/main/java/org/wikipedia/search/SearchResult.kt

```diff
@@ -0,0 +1,46 @@
+package org.wikipedia.search
+
+import org.wikipedia.dataclient.WikiSite
+import org.wikipedia.dataclient.mwapi.MwQueryPage
+import org.wikipedia.model.BaseModel
+import org.wikipedia.page.PageTitle
+
+class SearchResult : BaseModel {
+    val pageTitle: PageTitle
+    val redirectFrom: String?
+    val type: SearchResultType?
```

Comment:
This whole thing can be turned into a `data class`, with these three fields in the primary constructor.  Also, `type` should not be nullable.